src/main/java/emissary/place/MultiFileUnixCommandPlace.java

```diff
@@ -685,9 +682,6 @@ protected int processCommand(String[] cmd) {
         StringBuilder outbuf = new StringBuilder();
         StringBuilder errbuf = new StringBuilder();
         int status = processCommand(cmd, outbuf, errbuf);
-        if (outbuf.length() > 0 && logger.isDebugEnabled()) {
-            logger.debug("Discarding process stdout " + outbuf.toString());
-        }
         if (errbuf.length() > 0 && logger.isDebugEnabled()) {
             logger.debug("Discarding process stderr " + errbuf.toString());
```

Comment:
Are we sure the errbuf NEVER has anything we don't want to log?

Reply:
I left it because in the Executrix, the err is defined as "the destination to capture the standard error". From my work with the executrix in PcapPlaceTest, it usually puts out a file name and then a message regarding file format/type or the error (for example, /bin/sh: 1: BadCommand: not found). That being said, if you feel I should remove it, I can.

Reply:
I did not know of a scenario where this would be a problem, only feared that it COULD be a problem and by that was ok with just removing it to be sure.

Reply:
Just removed the errbuf logging.